pkg/server/routes.go

```diff
@@ -364,6 +364,11 @@ func (s *Server) routes(
 			store.FetchNotesBySystemIntakeID,
 			services.NewAuthorizeRequireGRTJobCode(),
 		),
+		services.NewCreateNote(
```

Comment:
If I were REALLY persnickety, I would say that this should be at the route /note, not /notes, since we're only creating a single note. But we're not fetching a single note, so I don't feel the need to separate these into two separate endpoints for now. It's pretty easy to do later if we are so compelled.

Reply:
I would say that they should all be under `notes`, but I can follow our convention for now and argue for a new one later.